project/Build.scala

```diff
@@ -776,7 +779,7 @@ object Build {
         baseDirectory.value.getParentFile / "test-common/src/main/scala",
       unmanagedSourceDirectories in Test +=
         baseDirectory.value.getParentFile / "test-common/src/test/scala"
-  ).dependsOn(jsEnvs)
+  ).dependsOn(io, jsEnvs)
```

Comment:
If we move `escapeJS` to `js-envs`, then this change might not be needed after all.